tests/GraphQL/WalletPaymentTransactionsTest.php

```diff
@@ -65,23 +65,22 @@ public function testItInitiatesContributionPlanWithdrawalTransactionSuccessfully
     /**
      * @test
      */
-    public function testItCorrectlyApprovesAContributionWithdrawalTransaction()
+    public function testItCorrectlyApprovesAWalletPaymentTransaction()
```

Comment:
Where is the code that actually tests it it approves? All this does is to create entries in the database